src/locales/cs.ts

```diff
@@ -0,0 +1,798 @@
+    john_fisher_bishop_and_thomas_more_martyrs: 'Sv. Jana Fishera, biskupa a Tomáše Mora, mučedníků',
+  //   louis_versiglia_bishop_and_callistus_caravario_priest_martyrs: 'Saints Louis Versiglia, Bishop and Callistus Caravario, Priest, Martyrs',
+    marcellinus_of_rome_and_peter_the_exorcist_martyrs: 'Sv. Marcelina a Petra, mučedníků',
+  //   margaret_clitherow_anne_line_and_margaret_ward_martyrs: 'Saints Margaret Clitherow, Anne Line and Margaret Ward, Virgin, Martyrs',
+  //   marko_krizin_melchior_grodziecki_and_stephen_pongracz_priests: 'Saints Marko Krizin, Melchior Grodziecki and Stephen Pongrácz, Priests and Martyrs',
+    michael_gabriel_and_raphael_archangels: 'Sv. Michaela, Gabriela a Rafaela, archandělů',
+    nereus_of_terracina_and_achilleus_of_terracina_martyrs: 'Sv. Nerea a Achillea, mučedníků',
+  //   mary_soledad_torres_acosta_virgin: 'Saint Mary Soledad Torres y Acosta, Virgin',
+    // peter_baptist_blasquez_paul_miki_and_companions_martyrs: 'Sv. Peter Baptist Blásquez, Pavla Mikiho a druhů, mučedníků',  // TODO: Localise `Peter Baptist Blásquez`
+    perpetua_of_carthage_and_felicity_of_carthage_martyrs: 'Sv. Perpetuy a Felicity, mučednic',
+    philip_and_james_apostles: 'Sv. Filipa a Jakuba, apoštolů',
+  //   philip_evans_and_john_lloyd_priests: 'Saints Philip Evans and John Lloyd, Priests and Martyrs',
+    pontian_i_pope_and_hippolytus_of_rome_priest: 'Sv. Ponciána, papeže, a Hippolyta, kněze, mučedníků',
+  //   spyridon_of_trimythous_bishop: 'Saint Spyridon, Bishop',
+  //   roch_gonzalez_alphonsus_rodriguez_and_john_del_castillo_priests: 'Saints Roch González, Alphonsus Rodríguez, and John del Castillo, Priests and Martyrs',
+  //   rupert_of_salzburg_and_virgilius_of_salzburg_bishops: 'Saints Rupert and Virgilius of Salzburg, Bishops',
+    simon_and_jude_apostles: 'Sv. Šimona a Judy, apoštolů',
+    stanislaus_of_szczepanow_bishop: 'Sv. Stanislava, biskupa a mučedníka',
+  //   stanislaus_kazimierczyk_priest: 'Saint Stanislaus Kazimierczyk, Priest',
+  //   stanislaus_kostka_religious: 'Saint Stanislaus Kostka, Religious',
+    stephen_i_of_hungary: 'Sv. Štěpána Uherského',
+    stephen_the_first_martyr: 'Sv. Štěpána, prvomučedníka',
+    timothy_of_ephesus_and_titus_of_crete_bishops: 'Sv. Timoteje a Tita, biskupů',
+  //   sunniva_of_norway_virgin: 'Saint Sunniva, Virgin and Martyr',
+  //   swithun_of_winchester_bishop: 'Saint Swithun, Bishop',
+    sylvester_i_pope: 'Sv. Silvestra I. papeže',
+  //   teilo_of_llandaff_bishop: 'Saint Teilo, Bishop',
+    teresa_benedicta_of_the_cross_stein_virgin: 'Sv. Terezie Benedikty od Kříže, panny a mučednice',
+    teresa_of_calcutta_religious: 'Sv. Terezy z Kalkaty, řeholnice',
+  //   teresa_of_jesus_jornet_ibars_virgin: 'Saint Teresa of Jesus Jornet Ibars, Virgin',
+    teresa_of_jesus_of_avila_virgin: 'Sv. Terezie od Ježíše z Ávily, panny a učitelky církve',
```

Comment:
I understand the combined title "od Ježíše z Ávily" is your editorial decision (I don't remember where exactly, but it was discussed somewhere quite recently), but *noone ever* calls her like this in Czech Republic. It looks and sounds very strange. Either "Sv. Terezie od Ježíše", or "Sv. Terezie z Ávily", never both.

Reply:
True, it is my editorial decision, because we (in romcal) try to support all locales and all calendars, and as a matter of fact, there are multiple (at least three) St Teresias with the name `St Teresia of [Child] Jesus`. IMO, we need to be able to exactly know of which Teresia is the celebration named.

> but _noone ever_ calls her like this in Czech Republic.

Same here, in Slovakia and I think anywhere else, _where no other St Teresia of Jesus is celebrated_.